rsconnect/main.py

```diff
@@ -821,6 +827,105 @@ def deploy_manifest(name, server, api_key, insecure, cacert, new, app_id, title,
     )
 
 
+@server_args
+@content_args
+@click.option(
+    "--entrypoint",
+    "-e",
+    help=("The module and executable object which serves as the entry point for the deployment."),
+)
+@click.option(
+    "--exclude",
+    "-x",
+    multiple=True,
+    help=(
+        "Specify a glob pattern for ignoring files when building the bundle. Note that your shell may try "
+        "to expand this which will not do what you expect. Generally, it's safest to quote the pattern. "
+        "This option may be repeated."
+    ),
+)
+@click.argument("directory", type=click.Path(exists=True, dir_okay=True, file_okay=False))
+@click.argument(
+    "extra_files",
+    nargs=-1,
+    type=click.Path(exists=True, dir_okay=False, file_okay=True),
+)
+@click.argument("path", type=click.Path(exists=True, dir_okay=True, file_okay=True))
+def deploy_html(
+    name,
+    server,
+    api_key,
+    insecure,
+    cacert,
+    new,
+    app_id,
+    title,
+    verbose,
+    path,
+    env_vars,
+    entrypoint,
+    extra_files,
+    excludes,
+):
+    set_verbosity(verbose)
+
+    entrypoint = entrypoint or infer_entrypoint(path, "text/html")
+    manifest = make_html_manifest(entrypoint)
+
+    with cli_feedback("Checking arguments"):
+        connect_server = _validate_deploy_to_args(name, server, api_key, insecure, cacert)
+        manifest = validate_manifest_file(manifest)
```

Comment:
This seems to be a leftover from `deploy manifest`. The manifest file is known to be good because rsconnect-python just created it (user-provided manifests need validation when using `deploy manifest`).